### ISSUE TITLE
How do I get the reconstructed MRI images?

### ISSUE THREAD

**Issue by lzl2040** [JustlfC03/TC-DiffRecon#2]
In ```image_sample_complex_duo.py```, the amplitude of the frequency is saved as a picture. Could you tell me how to get the reconstructed MRI image? Is it the inverse Fourier transform of ```sample2```? Thank you!

**Comment by JustlfC03** [comment]
I don't quite understand what you mean, what you get after model sampling `image_sample_complex_duo.py` is the reconstructed MRI image. Are you talking about k-space images? k-space image is obtained by MRI after inverse Fourier transform.

**Comment by lzl2040** [comment]
> I don't quite understand what you mean, what you get after model sampling `image_sample_complex_duo.py` is the reconstructed MRI image. Are you talking about k-space images? k-space image is obtained by MRI after inverse Fourier transform.

Thank you for your reply. I have  run 20,000 steps with batch size 2 and run ```image_sample_complex_duo.py```, but the reconstructed MRI image is as follows:
![file1000033_16 (1)](https://github.com/user-attachments/assets/6f494b0d-c82a-40f9-a73a-459d0b5f5b70)

Is this because the training steps are not enough? I find that the loss is already very small.

**Comment by JustlfC03** [comment]
I don't think so. I think you've entered the data in the wrong format.

**Comment by lzl2040** [comment]
When I downloaded the fastMRI dataset from the OneDrive, I followed ```readme.md``` to first run ```data_process.py```, then run ```image_train.py```.
![2](https://github.com/user-attachments/assets/ce145f66-fe37-441f-a7b9-584c9bf4ff17)

**Comment by JustlfC03** [comment]
I know where you are wrong, you should have misunderstood my `data_process.py` code, the comment code misled you. I now re-update the code, you can re-download this code to try again.

**Comment by lzl2040** [comment]
Thank you so much!
